lib/galaxy/managers/workflows.py

```diff
@@ -90,9 +90,14 @@ def check_security(self, trans, has_workflow, check_ownership=True, check_access
         if not check_ownership or check_accessible:
```

Comment:
Alright, I fixed the failing tests with https://github.com/galaxyproject/galaxy/pull/4465/commits/64f8e74adeb5f46e781fb3cf54294097c6301880. 

But the question remains, should you be able to construct subworkflows with workflows that you don't own ? I think the intention of the code was no, but the implementation was yes (even if you had no right to access the workflow ...). So should we allow this at least for workflows that the user can access? ping @jmchilton